<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LC_NamedViewsListWidget</class>
 <widget class="QWidget" name="LC_NamedViewsListWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <property name="horizontalSpacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="tbAddView">
       <property name="toolTip">
        <string>Create vew for current zoom and offset of drawing</string>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../../res/icons/icons.qrc">
         <normaloff>:/icons/nview_add.lci</normaloff>:/icons/nview_add.lci</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbRestoreView">
       <property name="toolTip">
        <string>Restores selected view</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../../res/icons/icons.qrc">
         <normaloff>:/icons/nview_visible.lci</normaloff>:/icons/nview_visible.lci</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbUpdateView">
       <property name="toolTip">
        <string>Update selected view by current zoom/position</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../../res/icons/icons.qrc">
         <normaloff>:/icons/zoom_auto.lci</normaloff>:/icons/zoom_auto.lci</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbRenameView">
       <property name="toolTip">
        <string>Edit selected view name</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../../res/icons/icons.qrc">
         <normaloff>:/icons/rename_active_block.lci</normaloff>:/icons/rename_active_block.lci</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbRemoveView">
       <property name="toolTip">
        <string>Removes selected view</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../res/icons/icons.qrc">
         <normaloff>:/icons/remove.lci</normaloff>:/icons/remove.lci</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QTableView" name="tvTable">
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <layout class="QHBoxLayout" name="layFilter">
     <item>
      <widget class="QLineEdit" name="leFilterMask">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Filtering views by name</string>
       </property>
       <property name="placeholderText">
        <string>Filter...</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="tbSettings">
     <property name="toolTip">
      <string>Settings</string>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../../res/icons/icons.qrc">
       <normaloff>:/icons/settings.lci</normaloff>:/icons/settings.lci</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../res/icons/icons.qrc"/>
  <include location="../../../../res/icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
